assets/js/blocks/active-filters/frontend.js

```diff
@@ -19,8 +19,8 @@ const getProps = ( el ) => {
 	};
 };
 
-renderFrontend(
-	'.wp-block-woocommerce-active-filters',
-	withRestApiHydration( Block ),
-	getProps
-);
+renderFrontend( {
```

Comment:
Passing an object instead of individual arguments since the renderFrontend method keeps growing.